Assets/BossRoom/Scripts/Server/ServerTestingHotkeys.cs

```diff
@@ -50,12 +50,12 @@ private void Update()
             if (m_SpawnEnemyKeyCode != KeyCode.None && Input.GetKeyDown(m_SpawnEnemyKeyCode))
             {
                 var newEnemy = Instantiate(m_EnemyPrefab);
-                newEnemy.SpawnWithOwnership(NetworkManager.Singleton.LocalClientId);
```

Comment:
Not a comment on this PR, but it would resolve an oversight on the previous PR. As part of guidelines, we are not allowed to be using #if UNITY_EDITOR preprocessor directives in the project. And as far as testing goes, it's okay for users to keep this in their release builds so let's eliminate line 14 & 66. Thanks!

Reply:
ok